src/components/OrderBookPanel/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// export interface TabItemType {
+//   readonly id: number
+//   title: string
+//   content: string
+//   readonly activeColor?: string
+// }
+
+// const tabItems: TabItemType[] = [
+//   {
+//     id: 1,
+//     title: 'BUY',
+//     content: '- buy component -',
+//     activeColor: '--color-buy',
+//   },
+//   {
+//     id: 2,
+//     title: 'SELL',
+//     content: '- sell component -',
+//     activeColor: '--color-sell',
+//   },
+// ]
+
+const Wrapper = styled.div`
+  display: flex;
+  padding: var(--padding-container-default);
+  background: none;
```

Comment:
Why?I don't see it inheriting background

Reply:
I guess defensive CSS syntax?